<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1235</width>
    <height>675</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="openGLScreen_groupBox">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>881</width>
      <height>661</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="title">
     <string>task_2</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout"/>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>10</y>
      <width>331</width>
      <height>661</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>311</width>
       <height>81</height>
      </rect>
     </property>
     <property name="title">
      <string>Settings</string>
     </property>
     <widget class="QSlider" name="f_slider">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>55</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>9</number>
      </property>
      <property name="pageStep">
       <number>1</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="f_lbl">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>43</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Factor</string>
      </property>
     </widget>
     <widget class="QLabel" name="f_num">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>30</y>
        <width>16</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox">
      <property name="geometry">
       <rect>
        <x>183</x>
        <y>50</y>
        <width>71</width>
        <height>26</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>9999</number>
      </property>
      <property name="value">
       <number>1</number>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>88</y>
       <width>444</width>
       <height>647</height>
      </rect>
     </property>
     <property name="title">
      <string>Light</string>
     </property>
     <widget class="QGroupBox" name="groupBox_7">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>381</y>
        <width>171</width>
        <height>168</height>
       </rect>
      </property>
      <property name="title">
       <string>Spot light</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="2" column="1">
        <widget class="QDoubleSpinBox" name="sp_pos_x">
         <property name="minimum">
          <double>-50.000000000000000</double>
         </property>
         <property name="maximum">
          <double>50.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="sp_pos_lb_3">
         <property name="text">
          <string>z</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QLabel" name="sp_pos_lb_11">
         <property name="text">
          <string>yd</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QLabel" name="sp_pos_lb_12">
         <property name="text">
          <string>zd</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="sp_I">
         <property name="maximum">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QDoubleSpinBox" name="sp_pos_z">
         <property name="minimum">
          <double>-50.000000000000000</double>
         </property>
         <property name="maximum">
          <double>50.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="sp_pos_lb_13">
         <property name="text">
          <string>f</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="sp_pos_lb_2">
         <property name="text">
          <string>y</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="sp_pos_lb">
         <property name="text">
          <string>x</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="sp_I_lb">
         <property name="text">
          <string>I</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="sp_pos_lb_10">
         <property name="text">
          <string>xd</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QDoubleSpinBox" name="sp_pos_y">
         <property name="minimum">
          <double>-50.000000000000000</double>
         </property>
         <property name="maximum">
          <double>50.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="5">
        <widget class="QPushButton" name="spot_col_btn">
         <property name="text">
          <string>color</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QDoubleSpinBox" name="sp_dir_x">
         <property name="minimum">
          <double>-1.000000000000000</double>
         </property>
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QDoubleSpinBox" name="sp_dir_y">
         <property name="minimum">
          <double>-1.000000000000000</double>
         </property>
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
        </widget>
       </item>
       <item row="4" column="3">
        <widget class="QDoubleSpinBox" name="sp_dir_z">
         <property name="minimum">
          <double>-1.000000000000000</double>
         </property>
         <property name="maximum">
          <double>1.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QDoubleSpinBox" name="sp_phi">
         <property name="minimum">
          <double>0.000000000000000</double>
         </property>
         <property name="maximum">
          <double>1.500000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>207</y>
        <width>171</width>
        <height>168</height>
       </rect>
      </property>
      <property name="title">
       <string>Point light</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <widget class="QLabel" name="pt_I_lb">
         <property name="text">
          <string>I</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="6">
        <widget class="QPushButton" name="pt_col_btn">
         <property name="text">
          <string>color</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="sp_pos_lb_6">
         <property name="text">
          <string>x</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="sp_pos_lb_5">
         <property name="text">
          <string>z</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="sp_pos_lb_4">
         <property name="text">
          <string>y</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QDoubleSpinBox" name="pt_pos_z">
         <property name="minimum">
          <double>-50.000000000000000</double>
         </property>
         <property name="maximum">
          <double>50.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QDoubleSpinBox" name="pt_pos_y">
         <property name="minimum">
          <double>-50.000000000000000</double>
         </property>
         <property name="maximum">
          <double>50.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QDoubleSpinBox" name="pt_pos_x">
         <property name="minimum">
          <double>-50.000000000000000</double>
         </property>
         <property name="maximum">
          <double>50.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="pt_I">
         <property name="maximum">
          <number>100</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>188</x>
        <y>32</y>
        <width>111</width>
        <height>517</height>
       </rect>
      </property>
      <property name="title">
       <string>Components</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="2" column="0">
        <widget class="QPushButton" name="amb_col_btn">
         <property name="text">
          <string>c</string>
         </property>
        </widget>
       </item>
       <item row="2" column="4">
        <widget class="QPushButton" name="spec_col_btn">
         <property name="text">
          <string>c</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QSlider" name="diff_col_slider">
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QSlider" name="amb_col_slider">
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
         <property name="pageStep">
          <number>10</number>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="Diff">
         <property name="text">
          <string>D</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="diff_col_btn">
         <property name="text">
          <string>c</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QLabel" name="Spec">
         <property name="text">
          <string>S</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="Amb">
         <property name="text">
          <string>A</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QSlider" name="spec_col_slider">
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>32</y>
        <width>171</width>
        <height>169</height>
       </rect>
      </property>
      <property name="title">
       <string>Directional light</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_7">
       <item row="0" column="0">
        <widget class="QLabel" name="dir_I_lb">
         <property name="text">
          <string>I</string>
         </property>
        </widget>
       </item>
       <item row="12" column="0" colspan="4">
        <widget class="QPushButton" name="dir_col_btn">
         <property name="text">
          <string>color</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="sp_pos_lb_8">
         <property name="text">
          <string>y</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="sp_pos_lb_7">
         <property name="text">
          <string>z</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="sp_pos_lb_9">
         <property name="text">
          <string>x</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QDoubleSpinBox" name="dir_pos_z">
         <property name="minimum">
          <double>-50.000000000000000</double>
         </property>
         <property name="maximum">
          <double>50.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QDoubleSpinBox" name="dir_pos_y">
         <property name="minimum">
          <double>-50.000000000000000</double>
         </property>
         <property name="maximum">
          <double>50.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QDoubleSpinBox" name="dir_pos_x">
         <property name="minimum">
          <double>-50.000000000000000</double>
         </property>
         <property name="maximum">
          <double>50.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="dir_I">
         <property name="maximum">
          <number>100</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
